h2o-core/src/main/java/water/api/RequestServer.java

```diff
@@ -665,7 +665,7 @@ private static NanoResponse serveError(H2OError error) {
 
   private static NanoResponse redirectToFlow() {
     NanoResponse res = new NanoResponse(HTTP_REDIRECT, MIME_PLAINTEXT, "");
-    res.addHeader("Location", "/flow/index.html");
+    res.addHeader("Location", H2O.ARGS.context_path + "/flow/index.html");
```

Comment:
@mmalohlava do you mean it doesn't redirect from i.e. localhost:54321 to localhost:54321/H2O/flow/index.html? @micahstubbs pointed that out also, do we need this redirect? It does work when the user doesn't set the context path, then localhost:54321 will get redirected to localhost:54321/flow/index.html. Currently with the context_path parameter passed the redirect happens only when the user inputs localhost:54321/H2O -> localhost:54321/H2O/flow/index.html.

This is because I'm changing the jetty context path, so it doesn't handle `/` requests. Changing this is, probably, doable, but I'm not sure how necessary - my understanding was that if the user is using the context_path parameter (s)he is ok with having to type it into the URL.